client/my-sites/sidebar/index.jsx

```diff
@@ -556,13 +556,15 @@ export class MySitesSidebar extends Component {
 
 		let planLink = '/plans' + this.props.siteSuffix;
 
+		const isUpgraded =
```

Comment:
we should have this also consider Jetpack plans as well, maybe use something like [`isCurrentPlanPaid`](https://github.com/Automattic/wp-calypso/blob/master/client/state/sites/selectors/is-current-plan-paid.js) ?